packages/ui/components/auth.tsx

```diff
@@ -0,0 +1,95 @@
+import { AppConfig, UserSession, showConnect } from "@stacks/connect";
+// import { Storage } from '@stacks/storage';
+import { StacksMainnet, StacksTestnet } from "@stacks/network";
+import {
+  callReadOnlyFunction,
+  cvToJSON,
+  standardPrincipalCV,
+} from "@stacks/transactions";
+
+// Initialize Gaia hub permissions for the user
+const appConfig = new AppConfig(["store_write", "publish_data"]);
+
+// Set this to true if you want to use Mainnet
+// Using a boolean helps to switch the entire application to mainnet
+// or testnet with only changing a single value
+const isNetworkMainnet = true;
```

Comment:
This should be an environment variable.

Reply:
Following the docs here (https://nextjs.org/docs/basic-features/environment-variables).
Making .env.development or .env files isn't working and environment variables aren't loaded up.
Spent an hour trying to make them to work but no progress :(
Dunno, what am I doing wrong

Reply:
Did you create `.env.local` in `packages/ui`?

Reply:
Yes, I have created all types of `.env` files in `packages/ui` and have logged the object `process.env` in the console and it appears to be empty. The environment variable doesn't seem to get loaded :/